lib/alerts/reducers/alerts.js

```diff
@@ -33,43 +31,47 @@ export const defaultState = {
 }
 
 const alerts = (state: AlertsReducerState = defaultState, action): AlertsReducerState => {
-  let alerts, entities
   switch (action.type) {
     case 'SET_ALERT_VISIBILITY_SEARCH_TEXT':
-      return update(state, {filter: {searchText: {$set: action.text}}})
+      return update(state, {filter: {searchText: {$set: action.payload}}})
     case 'SET_ALERT_VISIBILITY_FILTER':
-      return update(state, {filter: {filter: {$set: action.filter}}})
+      return update(state, {filter: {filter: {$set: action.payload}}})
     case 'SET_ALERT_SORT':
-      return update(state, {filter: {sort: {$set: action.sort}}})
+      return update(state, {filter: {sort: {$set: action.payload}}})
     case 'SET_ALERT_AGENCY_FILTER':
-      return update(state, {filter: {feedId: {$set: action.feedId}}})
+      return update(state, {filter: {feedId: {$set: action.payload}}})
     case 'DELETE_ALERT':
     case 'REQUEST_RTD_ALERTS':
       return update(state, {
         isFetching: {$set: true},
         fetched: {$set: false},
         all: {$set: []}
       })
-    case 'RECEIVED_GTFS_STOPS_AND_ROUTES':
-      if (action.module !== 'ALERTS') {
+    case 'RECEIVED_GTFS_STOPS_AND_ROUTES': {
+      if (action.payload.moduleType !== 'ALERTS') {
+        // Do not process entities fetched for a different module.
+        return state
+      }
+      const entities = state.entities
```

Comment:
You may want to clone entities here as well or at least call update on each entity when you set the gtfs property.  Otherwise, data in the store is being directly edited.